client/src/components/Login.js

```diff
@@ -36,15 +38,21 @@ function Login(props) {
         props.updateLoggedIn(true);
         props.getName(results.data.name);
         props.getUserId(results.data.id)
+        console.log(results.data)
+      })
+      .catch((err) => {
+        if (err.response.data.message === "Login not successful") {
```

Comment:
while  you can check the error message, it's usually safer to check for an "error code" so that you can change the message without having to change your code (for example if you want to translate your app to another language)

Reply:
Thank you! I checked for error code 400 

```suggestion
        if (err.response.status === 400) {
```